docs/api/js/getting-started.md

```diff
@@ -48,13 +48,13 @@ const provider = new Provider("https://testnet.era.zksync.dev");
 const wallet = new Wallet("<WALLET-PRIVATE-KEY>").connect(provider);
 ```
 
-**Note:** Currently, only `goerli` network is supported.
+**Note:** Currently, `goerli` and `sepolia` network is supported.
```

Comment:
This should be rephrased, as its no longer singular and please capitalize the network names.